packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -8,5 +8,6 @@
   "force_source_push_default_scratch_org_text": "SFDX: Push Source to Default Scratch Org",
   "force_source_status_text": "SFDX: View All Changes (Local and in Default Scratch Org)",
   "force_apex_test_run_text": "SFDX: Invoke Apex Tests...",
-  "cancel_sfdx_command_text": "SFDX: Cancel Active Command"
+  "cancel_sfdx_command_text": "SFDX: Cancel Active Command",
+  "force_apex_class_create": "SFDX: Create Apex Class"
```

Comment:
Hi there. This is my last day before my PTO. But using the text from IDE 2, prefixed with "SFDX: ", sounds great to me.